repos/system_upgrade/el7toel8/actors/checklocalnetworkscripts/libraries/library.py

```diff
@@ -0,0 +1,26 @@
+import os
+from leapp import reporting
+
+
+def generate_report():
+    """ Generate reports informing user about possible manual intervention requirements """
+
+    DOC_URL = "https://red.ht/upgrading-RHEL-8-to-RHEL-9-network-scripts"
+
+    # Show documentation url if local scripts of network-scripts installed
+    title = "local scripts of network-scripts detected"
+    summary = "Upgrade will be stopped. Additional steps might be required.\n\n"
+
+    reporting.create_report(
+        [
+            reporting.Title(title),
+            reporting.Summary(summary),
+            reporting.Severity(reporting.Severity.HIGH),
+            reporting.Flags([reporting.Flags.INHIBITOR]),
+            reporting.Tags([reporting.Tags.SERVICES]),
+            reporting.ExternalLink(
+                title="Please migrate custom network-scripts to NetworkManager dispatcher scripts.",
+                url=DOC_URL,
+            ),
```

Comment:
Add please resources fields:
```
reporting.RelatedResource('file', </abs/path/to/file>)
```
You can mention all files you test for the existence (one per the resource always) or just those that really exists on the system - both solutions are ok. but add the field.

As well, I suggest to add `reporting.Remediation` with `hint` pointing people they have to migrate the local  network-scripts to NetworkManager dispatcher scripts following the official documentation.

Reply:
Resource fields and `reporting.Remediation` were added.